<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REDTabSettingsDateTime</class>
 <widget class="QWidget" name="REDTabSettingsDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>424</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>RED Brick</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="1">
                 <widget class="QDateEdit" name="time_date_red">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="displayFormat">
                   <string>dd. MMMM yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Time:</string>
                  </property>
                  <property name="buddy">
                   <cstring>time_time_red</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Date:</string>
                  </property>
                  <property name="buddy">
                   <cstring>time_date_red</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QTimeEdit" name="time_time_red">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="displayFormat">
                   <string>HH:mm:ss</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="time_timezone_red">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <property name="currentText">
                   <string/>
                  </property>
                  <property name="insertPolicy">
                   <enum>QComboBox::NoInsert</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="save_timezone_button">
                  <property name="text">
                   <string>Save Timezone</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_33">
                  <property name="text">
                   <string>Timezone:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Local</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="2" column="0">
                 <widget class="QLabel" name="label_36">
                  <property name="text">
                   <string>Timezone:</string>
                  </property>
                  <property name="buddy">
                   <cstring>time_timezone_local</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_34">
                  <property name="text">
                   <string>Date:</string>
                  </property>
                  <property name="buddy">
                   <cstring>time_date_local</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_35">
                  <property name="text">
                   <string>Time:</string>
                  </property>
                  <property name="buddy">
                   <cstring>time_time_local</cstring>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="time_timezone_local">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Collecting data...</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDateEdit" name="time_date_local">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="displayFormat">
                   <string>dd. MMMM yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QTimeEdit" name="time_time_local">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="displayFormat">
                   <string>HH:mm:ss</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="time_sync_button">
                  <property name="text">
                   <string>Synchronize Brick with Local</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="red_reconfigure_label">
           <property name="text">
            <string>The RED Brick's timezone configuration is missing or incomplete. Please select a zone or synchronize with local time configuration.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>time_date_red</tabstop>
  <tabstop>time_date_local</tabstop>
  <tabstop>time_time_red</tabstop>
  <tabstop>time_time_local</tabstop>
  <tabstop>time_timezone_local</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
